be/src/exec/vectorized/hdfs_scanner_orc.cpp

```diff
@@ -21,7 +21,19 @@ class ORCHdfsFileStream : public orc::InputStream {
 
     uint64_t getLength() const override { return _length; }
 
-    uint64_t getNaturalReadSize() const override { return 2 * 1024 * 1024; }
+    uint64_t getNaturalReadSize() const override { return 1 * 1024 * 1024; }
```

Comment:
why NaturalReadSize is 1M now but  1M in previous version? add some comment to NaturalReadSize

Reply:
Because paper in `Delta Lake: High-Performance ACID Table Storage over Cloud Object Stores` suggests that use 1MB when reading S3. It's natural read size of S3.